app/js/lib/foxbox/api.js

```diff
@@ -9,16 +9,16 @@ const p = Object.freeze({
 
   watchTimer: Symbol('watchTimer'),
   watchEventBus: Symbol('watchEventBus'),
-  watchGetters: Symbol('watchGetters'),
+  watchChannels: Symbol('watchChannels'),
 
   // Private methods.
   getURL: Symbol('getURL'),
   onceOnline: Symbol('onceOnline'),
   onceAuthenticated: Symbol('onceAuthenticated'),
   onceDocumentVisible: Symbol('onceDocumentVisible'),
   onceReady: Symbol('onceReady'),
-  fetchGetterValues: Symbol('fetchGetterValues'),
-  updateGetterValue: Symbol('updateGetterValue'),
+  fetchChannelValues: Symbol('fetchChannelValues'),
+  updateChannelValue: Symbol('updateChannelValue'),
```

Comment:
Yep, I agree that we should have consistent patters, I can rename `fetchChannelValues` to `getChannelValues`, but `updateChannelValue` is quite different from BaseService.set. BaseService.set sets the value for the particular channel (and change is persisted in the foxbox/device), while `updateChannelValue` just updates channel value in internal app cache, so I'd prefer to keep it `update` here.

Sounds good?